packages/components/grid/src/stories/selection.stories.ts

```diff
@@ -49,24 +67,17 @@ export const Multiple: Story = {
   args: {
```

Comment:
In Safari bulk actions are not centered

<img width="1744" alt="image" src="https://github.com/user-attachments/assets/a7f04676-cd01-4903-b072-1a352705b10f" />

Reply:
<img width="1740" alt="image" src="https://github.com/user-attachments/assets/24886b75-3ddb-446d-b06d-628e5b42b073" />

Reply:
Fixed